src/linker/Linker/LinkContext.cs

```diff
@@ -468,10 +468,84 @@ public bool IsOptimizationEnabled (CodeOptimizations optimization, AssemblyDefin
 
 		public void LogMessage (MessageContainer message)
 		{
-			if (LogMessages && message != MessageContainer.Empty)
```

Comment:
Why is it OK to not consider the `Empty` message anymore? I thought we used that for warning suppression, so how does it work without this?